android/properties.gradle.kts

```diff
@@ -41,3 +41,7 @@ val keystoreProperties = readProperties((project.properties["keystorePropertiesF
 requiredKeystoreProperties.forEach { property ->
   project.extra.set(property, keystoreProperties.getProperty(property,"sample_" + property))
 }
+
+// set APP_ID property
+val appIdProperty = "APP_ID"
+project.extra.set(appIdProperty, localProperties.getProperty(appIdProperty,  """""""" ))  // set empty string if property is null
```

Comment:
We can re-use the implementation for setting the required gradle properties by adding `OPENSRP_APP_ID` to this list `requiredFhirProperties`

Reply:
Then update the implementation here to default to empty String if `OPENSRP_APP_ID` is missing in the local.properties. Otherwise from the logic with the update it set the property to `sample_OPENSRP_APP_ID` which is not intended. Empty means no application ID was provided so we can proceed to display the settings page.

```kotlin
requiredFhirProperties.forEach { property ->
  project.extra.set(property, localProperties.getProperty(property, when {
    property.contains("URL") -> "https://sample.url/fhir/"
    property.equals("OPENSRP_APP_ID") -> ""
    else -> "sample_" + property
  }
  ))
}
```